core/store/LevelDBStore/LevelDBStore.go

```diff
@@ -433,6 +403,26 @@ func (bd *LevelDBStore) GetBlock(hash Uint256) (*Block, error) {
 
 func (bd *LevelDBStore) persist(b *Block) error {
 
+	unspents := make(map[Uint256][]uint16)
+	///////////////////////////////////////////////////////////////
+	// Get Unspents for every tx
+	unspentprefix := []byte{byte(IX_Unspent)}
+	for i:=0;i<len(b.Transactions); i++ {
+		txhash := b.Transactions[i].Hash()
```

Comment:
keep the coherently  style  change
for i:=0;i<len(b.Transactions); i++ {
to 
for i := 0; i < len(b.Transactions); i++ {

Keep the space before and after all operate symbol